tests/ztest/custom_output/src/main.c

```diff
@@ -0,0 +1,39 @@
+/*
+ * Copyright (c) 2016 Intel Corporation
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <ztest.h>
+
+static void test_assert_pass(void)
+{
+	ztest_test_pass();
+}
+
+static void test_assert_fail(void)
+{
+	ztest_test_fail();
```

Comment:
if this test failing it cause whole shippable to failing - this can be something rather for sample

Reply:
I removed this and kept only the passing and skipped tests---that is enough to show the new output anyway.